src/scheduler/FunctionCallServer.cpp

```diff
@@ -1,110 +1,176 @@
 #include <faabric/scheduler/FunctionCallServer.h>
```

Comment:
This diff is painful to watch. You can examine the source [here](https://github.com/faasm/faabric/blob/zeromq/src/scheduler/FunctionCallServer.cpp).